app/boards/shields/splitkb_aurora_sofle/splitkb_aurora_sofle.conf

```diff
@@ -0,0 +1,9 @@
+# Uncomment these two line to add support for encoders to your firmware
+# CONFIG_EC11=y
+# CONFIG_EC11_TRIGGER_GLOBAL_THREAD=y
+
+# Uncomment the following line to enable the OLED Display
+# CONFIG_ZMK_DISPLAY=y
+
+# Uncomment the following lines to enable RGB underglow
+# CONFIG_ZMK_RGB_UNDERGLOW=y
```

Comment:
It doesn't need `# CONFIG_WS2812_STRIP=y`?

Reply:
No, see `Kconfig.defconfig` where we do:

```
config ZMK_RGB_UNDERGLOW
    select WS2812_STRIP
    select SPI
```